v_5_2_0/master_template.go

```diff
@@ -382,12 +397,25 @@ storage:
         source: "data:text/plain;base64,{{ index .Files "conf/trusted-user-ca-keys.pem" }}"
 
     {{- if not .DisableCalico }}
+    {{- if eq .Cluster.Kubernetes.CloudProvider "aws" }}
+    - path: /srv/aws-cni.yaml
+      filesystem: root
+      mode: 0644
+      contents:
+        source: "data:text/plain;charset=utf-8;base64,{{  index .Files "k8s-resource/aws-cni.yaml" }}"
```

Comment:
I think this file might be missing from this PR. I was working on `v_6_0_0` which was copied from `v_5_2_0` but the file is not anywhere:
```
~/go/src/github.com/giantswarm/k8scloudconfig $ git checkout master
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
~/go/src/github.com/giantswarm/k8scloudconfig $ git pull
Already up to date.
~/go/src/github.com/giantswarm/k8scloudconfig $ find . -name aws-cni.yaml 
~/go/src/github.com/giantswarm/k8scloudconfig $ 
```